common/math/geometry/include/geometry/solver/polynomial_solver.hpp

```diff
@@ -84,6 +84,11 @@ class PolynomialSolver
  */
   double linearFunction(double a, double b, double t) const;
 
+  /**
+   * @brief Hard coded parameter, tolerance of calculation results of the PolynomialSolver
+   */
+  constexpr static double tolerance = 1e-7;
```

Comment:
You should describe why this value was determined.
For example, "This value was determined by @hakuturu583 based on personal experience and not on any specific technical basis."